libs/wingsdk/test/target-sim/__snapshots__/immutable-capture.test.ts.snap

```diff
@@ -124,19 +124,81 @@ if (!(await this.my_buckets[1].get(\\"foo\\") === \\"bar\\")) { throw new Error(
 }
 };
 })())({
-my_buckets: [(function(env) {
-  let handle = process.env[env];
+my_buckets: [(function() {
+  const handle = process.env.BUCKET_HANDLE_4fecd6d0;
   if (!handle) {
-    throw new Error(\\"Missing environment variable: \\" + env);
+    throw new Error(\\"Missing environment variable: BUCKET_HANDLE_4fecd6d0\\");
   }
-  return $simulator.findInstance(handle);
-})(\\"BUCKET_HANDLE_4fecd6d0\\"),(function(env) {
-  let handle = process.env[env];
+  const simulatorUrl = process.env.WING_SIMULATOR_URL;
+  if (!simulatorUrl) {
+    throw new Error(\\"Missing environment variable: WING_SIMULATOR_URL\\");
+  }
+
+  return (function makeSimulatorClient(url, handle) {
+    return new Proxy({}, {
+        get: function (_target, method, _receiver) {
```

Comment:
Any way we can avoid emitting this into all inflight code? Perhaps implement this in the sdk and call from here?

Reply:
I don't remember why but I think when I tried generating code that called `require("@winglang/sdk")` it didn't seem to like it. I can look into it again and create an issue

Reply:
So `require("@winglang/sdk")` still doesn't work, as the error I get from esbuild is like this:

```
test/target-sim/topic-producer.test.ts > publishing messages to topic
Error: Error: Failed to bundle function: Error: Build failed with 1 error:
../../../../../../private/var/folders/q3/0yxxhjr95vx4yfkk3lvy6dj40000gn/T/wingsdk.y2Spfd/.wing/function_c88517b0.js:24:17: ERROR: Could not resolve "@winglang/sdk"
    at createBundle (/Users/chrisr/dev/wing1/libs/wingsdk/src/shared/bundling.ts:40:11)
    at Sandbox.loadBundleOnce (/Users/chrisr/dev/wing1/libs/wingsdk/src/shared/sandbox.ts:98:32)
    at Sandbox.call (/Users/chrisr/dev/wing1/libs/wingsdk/src/shared/sandbox.ts:110:5)
    at Object.withTrace (/Users/chrisr/dev/wing1/libs/wingsdk/src/simulator/simulator.ts:520:24)
    at /Users/chrisr/dev/wing1/libs/wingsdk/src/simulator/simulator.ts:268:26
```

I'm not totally sure, but this could just be an issue with the SDK unit tests, because `require("@winglang/sdk")` _does_ seem to work in `.wing/preflight.js`. But it's also possible we are using some tricks in the CLI to make that work.

Anyways, I think I've found a workaround for the time being

Reply:
Hmm maybe I could also use symlink shenanigans?...

Reply:
I think adding an `alias` to the esbuild invocation could work. Something like:

`alias: require.resolve("@winglang/sdk")`

Reply:
`require.resolve()` still didn't work for me but I was able to use `alias` to clean things up a bit